kerl

```diff
@@ -971,20 +971,40 @@ probe_pkgs() {
     fi
 
     if [ -n "${kpp}" ]; then
+        ncurses_installed=0
+        ncurses_failed=""
+
         log_build_entry "[packages] Found package declarations for your Linux distro: ${os_release_id}"
         for pkg in ${kpp}; do
             cmd=$(eval echo "\$_KPP_${os_release_id}_probe ${pkg}")
             probe=$(${cmd})
             eval "${probe}" >/dev/null 2>&1
             probe_res=$?
-            if [ "${probe_res}" != 0 ]; then
-                msg="[packages] Probe failed for ${pkg} (distro: ${os_release_id}): probe \"${probe}\" returned ${probe_res}"
-                log_build_entry "${msg}"
-                warn "${msg}"
-            else
-                log_build_entry "[packages] Probe success for ${pkg} (distro: ${os_release_id})!"
-            fi
+            case "${pkg}" in
+            *ncurses*)
```

Comment:
This ensures `ncurses-dev` apk and `ncurses-devel` rpm are also considered so we also skip the test at L1004 for those os's.